scripts/start.sh

```diff
@@ -32,26 +32,27 @@ function setup_mysql() {
     for f in /docker-entrypoint-initdb.d/*; do
         cat $f | mysql -uroot -proot --host ${SQLFLOW_MYSQL_HOST} --port ${SQLFLOW_MYSQL_PORT}
     done
-    # Grant all privileges to any remote hosts so that the sqlserver can be scaled into more than one replicas.
-    mysql -uroot -proot -e "GRANT ALL PRIVILEGES ON *.* TO 'root'@'' IDENTIFIED BY 'root' WITH GRANT OPTION;"
```

Comment:
I removed the `GRANT` command just because all the containers would be in the same Pod on k8s.